package.json

```diff
@@ -24,6 +24,7 @@
   "main": "index.js",
   "scripts": {
     "build": "tsc && gulp build:icons",
+    "exp": "tsc && gulp build:icons && pm2 stop n8n && pm2 start n8n",
```

Comment:
this one is the command to watch typescript code and restart n8n after, right? @moodyjmz

Reply:
![image](https://github.com/flagbit/akeneo-node-n8n/assets/61736610/ec5ea772-e88e-43e9-bc1a-837ea0027101)
command works !